adsmp/app.py

```diff
@@ -160,3 +217,59 @@ def get_msg_status(self, msg):
                 return 'active'
         else:
             return 'unknown'
+        
+    
+
+    def reindex(self, solr_docs, batch_insert, batch_update, solr_urls):
+        """Sends documents to solr and to Metrics DB.
```

Comment:
ok, i can add a parameter to choose what will be updated - but normally, i prefer to have them sent/updated at the same time